### PR TITLE
fix(helm): enable startup probe on server pod

### LINKED ISSUE #102: Livenessprove doesn't give enough time to pod
Since v1.106.1, the merge between server and microservice containers made the new server container a lot heavier and the livenessprobe  has a threshold too low, not giving enough time for the container to properly start. So the container enters a CrashLoopBackOff cicle.

It took me a lot longer to find out because in the beginning the container has very little log and I was assuming it was a bug in the newer version.

I intend to write a PR next week, just wanted to give a heads up as soon as I found out.

### PR DESCRIPTION
**Pull request by willianpaixao**
Fix #102

### CODE REVIEW

File: charts/immich/templates/server.yaml [hunk 1]
```diff
@@ -58,8 +58,10 @@ probes:
       timeoutSeconds: 1
       failureThreshold: 3
   readiness: *probes
-  startup:
-    enabled: false
+  startup: *probes
+    spec:
```
Comment (bo0tzz) [inline charts/immich/templates/server.yaml:62]: Best I can tell, this is not valid YAML, but somehow helm is letting it through anyways - but then the diff it produces is really weird & touches a bunch of unrelated stuff.

**Review by bo0tzz — CHANGES_REQUESTED** [review]

File: charts/immich/templates/server.yaml (hunk 1 above)
Comment (C-Otto) [reply charts/immich/templates/server.yaml:62]: https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/

Comment (willianpaixao) [reply charts/immich/templates/server.yaml]: This is how I've been using for some time, but couldn't find in the Helm documentation, so I fixed the PR.

**Review by bo0tzz — CHANGES_REQUESTED** [review]
This still doesn't seem to be working. The output I get is

```yaml
...
readinessProbe:
  failureThreshold: 3
  httpGet:
    path: /api/server-info/ping
    port: http
  initialDelaySeconds: 0
  periodSeconds: 10
  timeoutSeconds: 1
startupProbe:
  failureThreshold: 30
  initialDelaySeconds: 0
  periodSeconds: 10
  timeoutSeconds: 1
...
```

which is missing the `httpGet` block.

**Comment by willianpaixao** [comment]
@bo0tzz I had to drop the use of anchors and fully write the startup probe. Please take a look now.

**Review by bo0tzz — APPROVED** [review]

**Pull request merged**